templates/web.yml.template

```diff
@@ -70,6 +70,15 @@ httpClient:
   retries: 0
   
 logging:
+  loggers:
+    #customize logging for hibernate session metrics
+    "org.hibernate.engine.internal.StatisticalLoggingSessionEventListener":
+      # false = don't create new log event, but override existing
+      additive: false
```

Comment:
Why are you changing this? The default is true, isn't it?

I hadn't seen the setting before, so I'm probably not grokking.

Reply:
If you leave as default/true, you create another log event. So it'll have 2 log events: 1 log event with the old format and 1 log event with the new format.